### DISCUSSION TITLE
doris使用iceberg外表无法读取oss

### DISCUSSION THREAD

**Discussion by lmxcode** [apache/doris#14933] [A - General / Q&A]
doris使用iceberg外表无法读取oss，如何设置才能正常读取？

**Comment by divinerapier** [comment]
![cb362df6-bc16-4245-b366-8ac2047ad8bf](https://github.com/user-attachments/assets/0aee175b-3aa9-48b7-b847-a7a77db2c3ea)
是这个问题吗
